README.md

```diff
@@ -70,6 +70,18 @@ Compile & deploy the smart contract suite:
 make deploy-contracts
 ```
 
+### Running e2e tests
+
+Run:
+
+```bash
+run tooling-e2e-tests/full_scenario.sh
```

Comment:
let's maybe add the `make deps` and `make generate-contracts` commands that are commonly forgotten